src/Illuminate/Console/Application.php

```diff
@@ -55,11 +55,13 @@ public function __construct(Container $laravel, Dispatcher $events, $version)
      */
     public function call($command, array $parameters = [])
     {
-        $parameters['command'] = $command;
+        $parameters = collect($parameters)->prepend($command);
 
         $this->lastOutput = new BufferedOutput;
 
-        return $this->find($command)->run(new ArrayInput($parameters), $this->lastOutput);
+        $this->setCatchExceptions(false);
```

Comment:
Ahh, i see what you mean. 

Ill take a look now, however from memory i think thats a private property of the Symfony class and theres only a setter. Ill investigate.